src/generators/src/infra/git.js

```diff
@@ -1,26 +1,23 @@
-module.exports = async ({ template: { generate }, system, filesystem }) => async () => {
-  
-  process.stdout.write(`Generating git\n`)
+module.exports =
+  async ({ template: { generate }, system, filesystem }) =>
+    async () => {
+      const directory = filesystem.cwd()
+      const platform = process.platform
+      const { spawnSync } = require('child_process')
+      const where = platform === 'win32' ? 'where.exe' : 'which'
+      const out = spawnSync(where, ['git'])
 
-  const platform = process.platform
-  const { spawn } = require('child_process')
-  const where = platform === 'win32' ? 'where' : 'which'
-  const out = spawn(where, ['git'])
-
-  out.on('close', (code) => {
-    // eslint-disable-next-line no-console
-    console.log(`child process exited with code ${code}`)
-  })
-
-  const directory = filesystem.cwd()
-
-  await generate({
-    template: 'infra/config/.gitignore.ejs',
-    target: '.gitignore'
-  })
-
-  await system.run('git init', { cwd: directory })
-  await system.run('git branch -m main', { cwd: directory })
-  await system.run('git add --all', { cwd: directory })
-  await system.run('git commit -m "initial commit"', { cwd: directory })
-}
+      if (out.exitCode === 1) {
+        process.stdout.write(`Git not found\n`)
```

Comment:
maybe we could add more information here, like
`[ERROR] Git not found\n install it via https://git-scm.com/downloads and try again, or skip the git option when generates a new project with Herbs-CLI`